css/main.css

```diff
@@ -301,6 +301,65 @@ p.paragraph {
         margin: 10px 0 0 0;
     }
 }
+.whatisgulp {
+    padding: 90px 0;
+    background-color: #3A3E64;
+    color: rgba(255, 255, 255, 0.6);
+}
+.whatisgulp .listitem-heading {
+    color: rgba(255,255,255,0.9);
+}
+.whatisgulp .listitem-heading {
+    color: rgba(255,255,255,0.9);
+}
+.hljs-keyword {
+    color: rgba(195,154,78,1);
+}
+.hljs-string {
+    color: rgba(130 ,140,91,1);
+}
+.hljs-built_in {
+    color: rgba(112,75,40,1);
+}
+.editor-window .editor-menubar {
+    background-color: #E5E4E4;
+    height: 25px;
+    display: flex;
+    align-items: center;
+    border-radius: 5px 5px 0 0;
+}
+.editor-window code {
+    background-color: #141414;
+    font: normal 10pt monospace;
```

Comment:
@ansballard It looks like highlightjs is doing something weird with the font-family on strings - doesn't seem to be monospace.

Your call on svg vs. pre, btw. Can also use breakpoints to adjust font size so it looks alright on mobile and desktop.